docs/src/assets/scss/carbon-ads.scss

```diff
@@ -19,7 +19,7 @@
     background-color: var(--body-background-color);
     border: 1px solid var(--border-color);
     border-radius: var(--border-radius);
-    box-shadow: 0 1px 4px 1px hsla(0, 0%, 0%, 0.1);
+    box-shadow: 0 1px 4px 1px hsl(0 0% 0% / 0.1);
```

Comment:
we are using `hsla` for legacy reasons as some browsers dont support space syntax in `hsl` or `rgb`. 

more at https://stylelint.io/user-guide/rules/color-function-notation/

> **Note**
>Modern color-functions use a comma-free syntax because functions in CSS are used to group/name a syntax chunk. They should work by the same rules that CSS grammar does in general: values are optional and re-orderable when possible, space-separated, and commas are used to separate repetitions only.
For legacy reasons, rgb() and hsl() also supports an alternate syntax that separates all of its arguments with commas. Also for legacy reasons, the rgba() and hsla() functions exist using the same comma-based syntax.

Reply:
May be instead of removing it we can force option legacy? for all colors to use the comma syntax?

Reply:
OK, I forced the legacy option. I also when and made the other colour function rules explicit based on the current style